recipes/evidencemodeler/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -o errexit -o nounset
+
+readonly EVM_HOME=${PREFIX}/opt/${PKG_NAME}-${PKG_VERSION}
+
+mkdir -p ${EVM_HOME}
+
+cp -Rp simple_example PerlLib EvmUtils evidence_modeler.pl ${EVM_HOME}
```

Comment:
Shouldn't evidence_modeler.pl not be in PATH after installation? You could symlink it into PREFIX/bin